bump_packaging

```diff
@@ -12,7 +12,24 @@ fi
 REMOTE_BRANCH="${FLAVOR}/${FOREMAN_VERSION}"
 FULL_REMOTE_BRANCH="$PACKAGING_GIT_REMOTE/$REMOTE_BRANCH"
 
-cd "$PACKAGING_DIR"
+if [[ $GIT_USE_WORKTREES == true ]] ; then
+	RELEASE_PACKAGING_DIR="${PACKAGING_DIR}/${FLAVOR}-$VERSION"
+
+	if [[ ! -d "$RELEASE_PACKAGING_DIR" ]] ; then
+		(
+			cd "${PACKAGING_DIR}/master"
+			git worktree add "$RELEASE_PACKAGING_DIR" "$PACKAGING_GIT_REMOTE/$REMOTE_BRANCH"
+		)
+	fi
+else
+	RELEASE_PACKAGING_DIR="${PACKAGING_DIR}"
+
+	if [[ ! -d "$RELEASE_PACKAGING_DIR" ]] ; then
+		git clone --quiet -o "$PACKAGING_GIT_REMOTE" https://github.com/theforeman/foreman-packaging "${RELEASE_PACKAGING_DIR}"
```

Comment:
```suggestion
		git clone --quiet --origin "$PACKAGING_GIT_REMOTE" https://github.com/theforeman/foreman-packaging "${RELEASE_PACKAGING_DIR}"
```

Reply:
TIL there is actually a long option for this. Git isn't always consistent in having one.